From: Jan-Ivar Bruaroey <jan-ivar@users.noreply.github.com>
Date: Sat, 10 Feb 2024 17:08:58 -0500
Subject: (tmp-cherry-pick) Revert "Add the java proto options in metrics set
 proto." (c0eac979ca)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit f665f7faf4804cb1ce3c17ce67bfe8122b30d327.

Reason for revert: Speculative revert due to downstream breakages

Original change's description:
> Add the java proto options in metrics set proto.
>
> Bug: b/279024829
> Change-Id: Ib1604465dad1cd8b2b1198d53aa5d75191e56e2d
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/329220
> Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
> Commit-Queue: Manashi Sarkar <manashi@google.com>
> Cr-Commit-Position: refs/heads/main@{#41288}

Bug: b/279024829
Change-Id: I3329081de58f6e42c4bf1cb03c7d706cb3f12e64
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/329400
Auto-Submit: Björn Terelius <terelius@webrtc.org>
Owners-Override: Björn Terelius <terelius@webrtc.org>
Commit-Queue: Björn Terelius <terelius@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#41292}
---
 api/test/metrics/proto/metric.proto | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/api/test/metrics/proto/metric.proto b/api/test/metrics/proto/metric.proto
index d23b745a65..94921a57cb 100644
--- a/api/test/metrics/proto/metric.proto
+++ b/api/test/metrics/proto/metric.proto
@@ -12,9 +12,6 @@ syntax = "proto3";
 
 package webrtc.test_metrics;
 
-option java_multiple_files = true;
-option java_package = "org.webrtc.api.test.metrics.proto";
-
 // Root message of the proto file. Contains collection of all the metrics.
 message MetricsSet {
   repeated Metric metrics = 1;
